dimagi/pages/templates/sections/covid_19/applications/cdc.html

```diff
@@ -0,0 +1,24 @@
+{% extends "sections/covid_19/applications/_base.html" %}
+{% load i18n %}
+{% load dimagidotcom %}
+
+{% block icon %}
+     <div class="image cdc" data-lazybg="true"></div>
```

Comment:
inconsistent tab/ whitespace in this file.
Please go through this whole PR and make sure all indentation in `html`, `scss`, and `js` files are **two spaces**—python of course is **4 spaces**.

All other code quality rules have to be consistent with the rest of the application, including standards defined by lint files in the root directory. There are `.scss-lint.yml` and `.jshintrc` files available. Please add them to your IDE.

Also, from looking at this branch pushed to staging, it appears you have at least one unclosed div hanging around somewhere.